distributedsocialnetwork/post/retrieval.py

```diff
@@ -177,6 +177,61 @@ def get_detailed_post(post_id):
     # response = requests.get(
     #             url, headers={'content-type': 'application/json', 'Accept': 'application/json'})
 
+def get_comments(pk):
+    #return Comment.objects.filter(post_id=post_id)
+    comments = Comment.objects.filter(post_id=pk)
+    comm_ids = []
+    for comment in comments:
+        comm_ids.append(comment["id"])
+    nodes = list(Node.objects.all())
+    for node in nodes:
+        if node.node_auth_username != "":
+            # We can authenticate
+            url = node.api_url + 'posts/' + pk
```

Comment:
For better generic compatibility, we should be getting our Post object, getting it's origin, then generating the URL with that.

Also, the URL should be http://node-api-url-here/posts/{post_id}/comments, otherwise we are parsing the post again unnecessarily.